### ISSUE TITLE
Docker image with PDF signing

### ISSUE THREAD

**Issue by johnt-softclouds** [Keyfactor/signserver-ce#34]
Is there a SignServer-CE Docker image with the PDF signing plugin already installed?  If the answer is no, are there instructions on how to install the plugin using the SignServer-CE Docker image? Thank you.

**Comment by netmackan** [comment]
Yes, the official SignServer CE container image contains the PDF Signer (standard PDF signing not PAdES).
It is available on Docker Hub: https://hub.docker.com/r/keyfactor/signserver-ce

**Comment by netmackan** [comment]
(I will move this issue to Discussions instead at that is where we discuss questions)
